client/src/components/BottomBanner/index.tsx

```diff
@@ -0,0 +1,12 @@
+import BottomBanner from "./BottomBanner";
```

Comment:
discussion: should we put `BottomBanner.tsx` code in the `index.tsx` file here as a named export to avoid double exports 😄 ? 

e.g.,

```tsx
export const BottomBanner = connect(
  mapStateToProps,
  mapDispatchToProps
)(_BottomBanner);
```

Reply:
Yeah - lets do that for now! I wanted to suggest a new structure for the index files and folders but that is a bigger discussion and requires a little more time!  Thanks for the review!

Reply:
Sounds good thank you so much 🚀 ! Yeah would love to learn more about it once the launch is done 👍